thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/yaml/YamlResource.java

```diff
@@ -225,6 +267,7 @@ public Response createYamlAlert(@ApiParam(value =  "a json contains both notific
     }
     long alertId = Long.parseLong(((Map<String, String>) response.getEntity()).get("detectionAlertConfigId"));
 
+    createYamlOnboardingTask(detectionConfigId, startTime, endTime);
```

Comment:
Enclose in try-catch. If exception is thrown, revert changes and display message 'failed to setup the onboarding job`

Reply:
sure. good idea.